prac_09/cleanup_files.py

```diff
@@ -2,13 +2,22 @@
 CP1404/CP5632 Practical
 Demos of various os module examples
 """
-import shutil
 import os
 
 
 def get_fixed_filename(filename):
```

Comment:
does not work completely. code still adds spaces and underscores just inside brackets and continues to add extra underlines every time you run the program. try adding a check for brackets, spaces and underscores next to capitals that already exist.

Reply:
Thanks, all fixed now 👍